qiita_ware/test/test_ebi.py

```diff
@@ -243,6 +244,27 @@ def test_generate_curl_command(self):
                '%%20ebi_access_key%%3D"') % {'xml_dir': submission.xml_dir}
         self.assertEqual(obs, exp)
 
+    def test_parse_EBI_reply(self):
+        e = EBISubmission(2, 'ADD')
+        curl_result = ""
```

Comment:
Not sure is needed. The parser will return None in both cases, which are errors. Then commands.py should raise the error with the return of the command.